Intersect.Client/MonoGame/Graphics/MonoRenderer.cs

```diff
@@ -830,31 +820,25 @@ public void Init(GraphicsDevice graphicsDevice)
 
         public override GameFont LoadFont(string filename)
         {
-            //Get font size from filename, format should be name_size.xnb or whatever
+            // Get font size from filename, format should be name_size.xnb or whatever
             var name = GameContentManager.RemoveExtension(filename)
                 .Replace(Path.Combine("resources", "fonts"), "")
                 .TrimStart(Path.DirectorySeparatorChar);
 
+            // Split the name into parts
             var parts = name.Split('_');
-            if (parts.Length >= 1)
-            {
-                if (int.TryParse(parts[parts.Length - 1], out var size))
-                {
-                    name = "";
-                    for (var i = 0; i <= parts.Length - 2; i++)
-                    {
-                        name += parts[i];
-                        if (i + 1 < parts.Length - 2)
-                        {
-                            name += "_";
-                        }
-                    }
 
-                    return new MonoFont(name, filename, size, mContentManager);
-                }
+            // Check if the font size can be extracted
+            if (parts.Length < 1 || !int.TryParse(parts[parts.Length - 1], out var size))
+            {
+                return null;
             }
 
-            return null;
+            // Concatenate the parts of the name except the last one to get the full name
+            name = string.Join($"{'_'}", parts.Take(parts.Length - 1));
```

Comment:
Uh this should just be `'_'`, do not use an interpolated string.

Reply:
can't use it as char directly for string.Join

Reply:
![imagen](https://user-images.githubusercontent.com/17498701/210430773-0e90a137-74a9-4996-b060-497ef934530a.png)

other way around:
![imagen](https://user-images.githubusercontent.com/17498701/210430838-de93890d-152c-418b-a461-954107506e04.png)

Reply:
actually, `'_'` is used twice, i'll make a variable for it